docs/reports/tootsie-retro.md

```diff
@@ -0,0 +1,416 @@
+# Tootsie 8B Retrospective
+
+This is a retrospective on our first generation Marin 8B run, codenamed "Tootsie."
```

Comment:
I thought Tootsie referred to the general yolo vibes, not the 8B, because the 32B will also be Tootsie?  I think we can just call all future Marin runs tootsie runs.  We can just wordsmith this.